tests/ImportCommentsPreserved/import-comments-document-top-test3.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * This comment is before the whole block,
+ * it is not attached to the following import.
+ */
+
+import b from "b";
+import a from "a";
```

Comment:
How is this one different from `import-comments-document-top-test2?`

Reply:
It is not, I had other test cases that made this test file redundant, and I shall clean it up post-haste.